frontend/src/main/java/starter/project/frontend/AppConfig.java

```diff
@@ -0,0 +1,19 @@
+package starter.project.frontend;
+
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import starter.project.domain.Car;
+
+import java.util.HashMap;
+import java.util.Map;
+
+@Configuration
+public class AppConfig {
+    @Bean
+    public  Map<Integer, Car> map()
+    {
+        return  new HashMap<Integer, Car>() ;
```

Comment:
Doesn't need the types here. In IntelliJ you will see they are greyed out, that means it's useless.